docs/rfc-fluffy-unicorn.md

```diff
@@ -0,0 +1,152 @@
+# Nimona: Fluffy-Unicorn
+Fluffy-Unicorn is a self-describing, type-hinted, encoding-
+independent, language-agnostic, and consistently hash-able data format.
+(_It sounds fancier than it really is, I promise._)
+
+It does not define how data should be encoded or decoded when being transported. It can use any data format such as [JSON], [CBOR], [MsgPack], [Cap-n-proto], and others.
+
+Its purpose is to be able to define (relatively) strongly-typed and self-defined objects that can be transported through any means and encoding while still being able to produce the same cryptographic hash.
```

Comment:
Nice, but maybe expand with examples showing hashes of JSON in different languages to expose the need for FU?